### ISSUE TITLE
Story: Add Audio-Described File Link to MLPs

### ISSUE THREAD

**Issue by andyvanavery31** [NCIOCPL/cgov-digital-platform#4487]
### Specifications

**General**
- The addition of the audio-described file shall use a simple approach that does not change the video player itself
- The addition of the audio-described file shall assume that the video will no longer be a page-level content type
- The audio-described link paragraph will have been added to the CGDP Video Media Type prior to this ticket per #4484

**Drupal Specifications**
- The audio-described version of the video will be accessed through an external link
- The audio-described version of the video will be optional for videos
- If the video does not have a caption, and it has an audio-described link then the FigCaption should not be created, and the text and link for the audio-described link will not be a description for the video

**Front-End Specifications**
- The video component shall allow users to access the audio-described version of the video
- If the video does not have an audio described version to go with it, then the designed interface to go to the audio-described version will not render
- If the video has a caption, then the text and link to the audio-described link will appear in the caption
- If the video does not have a caption, then the audio-described link will appear alone

**Analytics**
- No analytics required. This component is to ensure videos are accessible. A low Click Through Rate (CTR) will not change or remove this feature.

### DoR Tasks

- [x] Content Model Drafted
- [x] Content Model Reviewed
  - [x] Developer Review
  - [x] IA Review
- [x] Content Model Approved
  - [x] Developer Approval
  - [x] IA Approval
- [x] Acceptance Criteria Drafted
- [x] Scenarios created for YAML
- [x] Acceptance Criteria Reviewed
  - [x] Developer Review
  - [x] IA Review
- [x] Acceptance Criteria Approved
  - [x] Developer Approval
  - [x] IA Approval

### DoD Tasks

- [ ] Governance Drafted
- [ ] PR Review (see PR for review and comments)
- [ ] Client review with demo

### Technical Notes

**Content Model**
- [Video Media Type](https://github.com/NCIOCPL/cgov-digital-platform/wiki/Media-Type-Video)

**Scenarios/Comps**
- On an example Mini Landing Page for test content, use the [DCEG Our Research Comp](https://www.figma.com/design/GCOniv8OTyj9OC1udKwo6S/Mini-Landing-Pages?node-id=8256-50062&node-type=frame&m=dev) as reference:
  - Page URL: Something similar to /test/video-ad-link/
  - Page Title: Test Page for Video AD Link
  - Components:
    - NCIDS 2 Column Layout
      - 2 Column Layout Display: 50/50
      - NCIDS Content Block: 10 sentences of lorem ipsum
      - NCIDS Video: Select a Featured Video with a caption and add an external link to the audio-described external link field for that video
    - NCIDS 2 Column Layout
      - 2 Column Layout Display: 50/50
      -  NCIDS Content Block
        - Body: 10 sentences of lorem ipsum
      - NCIDS Video: Select a Featured video without a caption and add an external link to the audio-described external link field for that video
    - NCIDS 2 Column Layout
      - 2 Column Layout Display: 66/33
      - NCIDS Content Block: 10 sentences of lorem ipsum
      - NCIDS Video: Select a Featured Video with a caption and add an external link to the audio-described external link field for that video
    - NCIDS 2 Column Layout
      - 2 Column Layout Display: 66/33
      -  NCIDS Summary Box
        - Title: Test Title
        - Body: 10 sentences of lorem ipsum
      - NCIDS Video: Select a Featured video without a caption and add an external link to the audio-described external link field for that video

- Repeat for Spanish content

### Acceptance Criteria

**CMS Experience - Adding an Embedded Video to a Mini Landing Page**
```gherkin
Scenario: A content editor adds a video to a Mini Landing Page using the 2 Column Layout
  Given the user is logged into the CMS as an editor
    And the user is adding a video to an MLP using the 2 Column Layout
  When the user is adding a video to an MLP using the 2 Column Layout
  Then the experience does not change whether the video has an audio-described link or not
```

**Front-End Experience**
```gherkin
Scenario: A user views a video with and audio-described link on a Mini Landing Page
  Given the audio-described link has been added to the video media type
    And the video with an audio-described link has been added to a Mini Landing Page
  When the video is added to the 2 Column Layout
    And it has a caption
  Then the audio-described link will appear below the caption with the text, "**Audio-Description:** Play the audio-described version of this video."
    And it will match the comp
  When the video is added to the 2 Column Layout
    And it does not have a caption
  Then the audio-described link will appear below the video with the text, "**Audio-Description:** Play the audio-described version of this video."
  When the user clicks on the link for the audio-described link
  Then they will be redirected to the external link for the audio-described video
```

### Related Tickets
* META: #4318
